plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -115,4 +114,9 @@ private static String convertAggregationExpressionToPql(AggregationExpression ag
     {
         return format("%s(%s)", aggregationExpression.getAggregationType(), aggregationExpression.getBaseColumnName()).toLowerCase(ENGLISH);
     }
+
+    public static String encloseInParentheses(String value)
+    {
+        return format("(%s)", value);
+    }
```

Comment:
Move this to a separate commit, as it seems unrelated to the main purpose of this commit.